ckanext/canada/templates/internal/recombinant/snippets/xls_upload.html

```diff
@@ -15,7 +15,7 @@
   {% set geno = h.recombinant_get_geno(pkg.dataset_type) %}
   {% if 'upload_warn' in geno %}
     <div class="alert alert-danger">
-      <p>{{ _(geno.upload_warn) }}</p>
+      <p>{{ _(geno.upload_warn) | safe }}</p>
```

Comment:
you want to insert HTML into the warning? That's ok, but seems unrelated to the PR